pandokia/helpers/runner_minipyt.py

```diff
@@ -234,10 +234,23 @@ def get_exception_str():
 
 def locate_test_methods(ob, test_order):
     l = []
+    f_names_found = []
 
-    # look through the class for methods that are interesting to us.
+    # Look through the class for methods that are interesting to us.
+    methodish_things = inspect.getmembers(ob, inspect.ismethod)
+    # For some reason in py3 inspect isn't finding our class tests methods
+    # in the HST ETC versions test class. adding the isfunction filter grabs
+    # them correctly and will not hurt to add.
```

Comment:
We talked about elaborating on this comment to explain it won't hurt to add because items on this list will be further inspected for the __test__ attribute.

Do we know *why* the inspect call is operating differently in Py3?

Reply:
No.  A cursory examination of The Web yielded nothing on the topic.  I could (and would like to) spend more time on it but so many other things to do.
 
Recall it might not be inspect but instead the way our class is written (which may somehow be nonstandard as far as inspect is concerned).